### PR TITLE
Dyn 5358 friendly splash screen

### PR DESCRIPTION
**Pull request by filipeotero**
### Purpose

This PR includes the following improvements for the splash screen:

* Close window before launching Dynamo
* Increase the size of the window

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@QilongTang @aabi

### FYIs

@RobertGlobant20 @jesusalvino

### CODE REVIEW

File: src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs [hunk 1]
```diff
@@ -89,6 +92,7 @@ internal async void SetLoadingDone()
             loadingTimer.Stop();
             loadingTimer = null;
             await webView.CoreWebView2.ExecuteScriptAsync($"window.setLoadingDone()");
+            await webView.CoreWebView2.ExecuteScriptAsync($"window.setTotalLoadingTime('Total loading time: {totalLoadingTime}ms')");
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs:95]: Do we need to localize this?

Comment (filipeotero) [reply src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs:95]: Yes, I will create a resource for it. Thanks!

**Review by QilongTang — APPROVED** [review]

File: src/DynamoSandbox/DynamoCoreSetup.cs [hunk 2]
```diff
@@ -67,7 +67,7 @@ public void RunApplication(Application app)
 
                 DynamoModel.RequestUpdateLoadBarStatus += DynamoModel_RequestUpdateLoadBarStatus;
 
-                splashScreen = new Dynamo.UI.Views.SplashScreen();
+                splashScreen = new Dynamo.UI.Views.SplashScreen(Program.DynamoCorePath);
```
Comment (QilongTang) [inline src/DynamoSandbox/DynamoCoreSetup.cs:70]: This does not seem the right path? FYI: @RobertGlobant20

Comment (RobertGlobant20) [reply src/DynamoSandbox/DynamoCoreSetup.cs:70]: @filipeotero the right path should be something like this:
C:\Users\myuser\AppData\Roaming\Dynamo\Dynamo Core\2.17\
Please verify if this is the expected path.

Comment (filipeotero) [reply src/DynamoSandbox/DynamoCoreSetup.cs:70]: Aaron is right, the path is wrong. But to get the user path, we need to load Dynamo model first. Is it ok @QilongTang?

Comment (QilongTang) [reply src/DynamoSandbox/DynamoCoreSetup.cs:70]: Hi @filipeotero If we load Dynamo Model first, then a few steps from the dynamic splash screen will be done by the time we display it. For now, would you hard code the path in CoreSetup?

Comment (filipeotero) [reply src/DynamoSandbox/DynamoCoreSetup.cs]: now its like this: `C:\Users\myuser\AppData\Roaming\Dynamo\Dynamo Core\2.17`

**Pull request merged**